build/build.xml

```diff
@@ -13,8 +13,9 @@
 		<mkdir dir="../temp-java"/>
 		
         <echo message="Using Java version ${ant.java.version}."/>
-
-        <javac source="1.6" srcdir="../jar/src" destdir="../temp-java" includeantruntime="false">
+        <echo message="android SDK ${android.sdk}."/>
+ 
+        <javac source="1.6" target="1.6" srcdir="../jar/src" destdir="../temp-java" includeantruntime="false">
```

Comment:
you need to specify target too so it builds with an old Java version that adt likes.